src/main/java/duke/task/Todos.java

```diff
@@ -0,0 +1,12 @@
+package duke.task;
+
+public class Todos extends Task {
```

Comment:
To keep it consistent as Event or Deadline, it might be better to rename this as Todo